src/themes/base/colors.ts

```diff
@@ -169,11 +169,13 @@ export const colorScheme: ColorSchemeMapping = _.mapValues(
       border: foreground,
       shadow: foreground,
       background: colorVariants[500],
+      lighterBackground: colorVariants[300],
```

Comment:
What is the purpose of lighterBackground? The other keys make sense to me. Should lighterBackground really just be a different value of background?

Reply:
`lighterBackground` was just something temporary I came up before we came up with a more robust color palette that would allow different shades of background|foreground|border|etc color for different UI states (active|hovered|focused|etc).

**tl;dr**
will remove it soon and add something like:

```typescript
background: {
  initial: colorVariants[500],
  hover: colorVariants[300],
},
```

Reply:
ah, ok, makes sense!

Reply:
not relevant anymore, see #756